.github/workflows/nix.yml

```diff
@@ -3,17 +3,33 @@ name: "CI - Nix"
 on:
   push:
     branches:
-      - master
       - devel
+    paths-ignore:
```

Comment:
We had an issue with that here: https://github.com/coal-library/coal/pull/705
Not sure if it sitll apply

Reply:
Something else was broken, because iirc the CI was just not running at all